### PR TITLE
Remove excess spacing for project cards

### PR DESCRIPTION
**Pull request by neelxie**
#### What does this PR do?

Remove excess spacing for project cards

#### Ticket
https://trello.com/c/YSadGvet

#### Screenshots
**Before**
Screenshot from 2021-06-16 11-02-48

**After**
Screenshot from 2021-06-16 15-16-38

### CODE REVIEW

**Comment by Mubangizi** [comment]
I think the card seems too wide, what do you say @neelxie

**Comment by neelxie** [comment]
> I think the card seems too wide, what do you say @neelxie
@Mubangizi i have fixed this, refer to updated image above.

**Review by Mubangizi — DISMISSED** [review]
Great work @neelxie, it looks good to me

File: src/components/AppsList/index.jsx [hunk 1]
```diff
@@ -50,6 +50,7 @@ class AppsList extends Component {
 
   render() {
     const { apps, isRetrieved, isRetrieving, params } = this.props;
+    console.log(apps);
```
Comment (MutegekiHenry) [inline src/components/AppsList/index.jsx:53]: @neelxie think you forgot to clean this up after usage

**Review by MutegekiHenry — COMMENTED** [review]
Great work @neelxie just a small clean up and all is good on my end.

**Comment by neelxie** [comment]
> Great work @neelxie just a small clean up and all is good on my end.

@MutegekiHenry this has been fixed

File: src/components/AppsList/index.jsx (hunk 1 above)
Comment (MutegekiHenry) [reply src/components/AppsList/index.jsx]: Great that its fixed now

**Review by MutegekiHenry — APPROVED** [review]

**Review by kawerewagaba — APPROVED** [review]
lgtm

**Pull request merged**